doc/users_guide/physics_processes.rst

```diff
@@ -7,7 +7,24 @@ The standard RAT simulation includes many standard GEANT4 physics processes.
 Physics Lists
 `````````````
 
-Add documentation for the Physics List!
+Add more documentation for the Physics List!
+
+Commands that related to the physics list are listed under ``/rat/physics``.
+You can find out what they do in ``src/cmd/src/PhysicsListMessenger.cc``.
+Descriptions to these commands will be added here soon. 
+
+
+Remove a process from the physics list
+''''''''''''''''''''''''''''''''''''''
+If you want to remove a process from the current physics list (e.g. if you
+would like to stop simulating muon decays), you can modify the physics list in
+your main macro file (e.g. run.mac) as follows:
+
+    /rat/physics/removeProcess mu+ Decay
+    /rat/physics/removeProcess mu- Decay

```

Comment:
Thanks. Please also document here how a user would find the name of the process.